scripts/download_nghttp3.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# If any commands fail, fail the script immediately.
+set -ex
+
+git clone --depth 1 --branch v1.1.0 https://github.com/ngtcp2/nghttp3 $1
```

Comment:
These versions should now be in the VERSIONS file so renovate can update them.

Reply:
I added ngtcp2, nghttp3 and quictls to the VERSIONS file now.